www/src/pages/foundations/elevation.jsx

```diff
@@ -0,0 +1,269 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import SEO from '../../components/SEO';
+import Layout from '../../components/PageLayout';
+import {
+  Button, Form, Container, Input, Toast,
+} from '~paragon-react'; // eslint-disable-line import/no-unresolved
+
+const boxShadowSides = ['down', 'up', 'right', 'left', 'centered'];
+const boxShadowLevels = [1, 2, 3, 4, 5];
+
+const controlsProps = [
+  { key: 'x', name: 'Offset X' },
+  { key: 'y', name: 'Offset Y' },
+  { key: 'blur', name: 'Blur' },
+  { key: 'spread', name: 'Spread' },
+  { key: 'color', name: 'Color' },
+];
+
+const BoxShadowNode = () => {
+  const [showToast, setShowToast] = useState(false);
+
+  const isBoxShadowCopied = (level, side) => {
+    navigator.clipboard.writeText(`@include pgn-box-shadow(${level}, "${side}");`);
+    setShowToast(true);
+  };
+
+  const boxShadowCells = boxShadowLevels.map(level => (
+    boxShadowSides.map(side => (
+      <div
+        key={side}
+        role="button"
+        className={`pgn-doc__box-shadow-cell box-shadow-${side}-${level}`}
+        tabIndex={0}
+        aria-label="Box-shadow cell"
+        onClick={() => isBoxShadowCopied(level, side)}
+        onKeyDown={(e) => {
+          if (e.key === 'Enter') {
+            isBoxShadowCopied(level, side);
+          }
+        }}
+      />
+    ))
+  ));
+
+  return (
+    <div className="pgn-doc__box-shadow-cells">
+      { boxShadowCells }
+      {(
+        <Toast
+          className="pgn-doc__box-shadow--toast"
+          onClose={() => setShowToast(false)}
+          show={showToast}
+          delay={2000}
+        >
+          Box-shadow copied to clipboard!
+        </Toast>
+      )}
+    </div>
+  );
+};
+
+const BoxShadowToolkit = ({ updateBoxShadow, id }) => {
+  const [boxShadowModel, setBoxShadowModel] = useState({
+    x: 10,
+    y: 10,
+    blur: 25,
+    spread: 0,
+    color: '#000',
+    inset: false,
+  });
+
+  const updateBoxShadowModel = (property, value) => {
```

Comment:
It might be worth adding a Segment event in this function to track when users interact with any of the fields? For example:

```
global.analytics.track('openedx.paragon.elevation.generator.updated', { property, value })
```

Reply:
Good idea, added